tests/integration/test_api.py

```diff
@@ -116,7 +115,7 @@ async def test_get_secrets__no_permissions(
         async with client.get(
             secrets_api.endpoint,
             headers=user.headers,
-            params={"project_name": project_name},
+            params={"org_name": "test-org", "project_name": project_name},
```

Comment:
there is an `org_name` fixture. can you use it please

Reply:
And since we are here, can you replace the "test-project" literal with the `project_name` fixture everywhere